cs/libdpr/src/FASTER.libdpr/dprfinder/enhanced/EnhancedDprFinderServer.cs

```diff
@@ -53,20 +63,29 @@ public void Dispose()
 
             termination.Set();
             processThread.Join();
-            backend.Dispose();
+            if(backend != null)
+            {
+                backend.Dispose();
+            }
+            if(clusterBackend != null)
+            {
+                // clusterBackend.Dispose();
```

Comment:
unintentional?

Reply:
There isn't a Dispose() method since there wasn't really much to dispose off. Also, this was used to run things without Kubernetes before deciding to put this functionality inside DprFinderBackend. ClusterBackend.cs will definitely get "deprecated" and I might reuse some of the code